app/alembic/versions/bb6536213ca6_add_invoice_delete_constraints.py

```diff
@@ -0,0 +1,24 @@
+"""Add invoice delete constraints
+
+Revision ID: bb6536213ca6
+Revises: 90f13c0c6117
+Create Date: 2021-08-24 19:15:39.352338
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'bb6536213ca6'
+down_revision = '90f13c0c6117'
```

Comment:
After deleting, you will need  to fix revision and down_revision fields in other files to fill gap created by this revision